common/build/eslint-plugin-fluid/src/test/mockFiles/no-unchecked-record-access/fileWithArrayAndRecordAccess.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const arr1 = [1, 2, 3];
+const value1 = arr1[0]; // This should not report an error
+
+const record = { a: 1, b: 2 };
+const uncheckedAccess = record["c"]; // This should report an error
```

Comment:
You need to remove this. It is not valid typescript. It can throw of checks.
That said you should have test cases with property access that does not raise an error.
  record["1"]  and record.b
should not problems.

Reply:
@RishhiB 

Can we also have positive cases (e.g., `const checkedAccess = record["a"]`) across the test cases?

Reply:
@jikim-msft yes please see common/build/eslint-plugin-fluid/src/test/mockFiles/no-unchecked-record-access/fileWithOnlyArrayAccess.ts